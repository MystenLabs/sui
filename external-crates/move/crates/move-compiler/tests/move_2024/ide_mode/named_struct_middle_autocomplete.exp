error[E03010]: unbound field
   ┌─ tests/move_2024/ide_mode/named_struct_middle_autocomplete.move:13:21
   │
13 │         let _tmp2 = _s.b.x;
   │                     ^^^^ Unbound field 'b' in 'a::m::B'

