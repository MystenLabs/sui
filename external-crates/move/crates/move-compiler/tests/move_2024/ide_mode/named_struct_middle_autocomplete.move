#[allow(ide_path_autocomplete)]
module a::m {

    public struct A has copy, drop {
        x: u64
    }

    public struct B has copy, drop {
        a: A
    }

    public fun foo() {
        let _s = B { a: A { x: 0 } };
        let _tmp2 = _s.b.x;
    }
}
