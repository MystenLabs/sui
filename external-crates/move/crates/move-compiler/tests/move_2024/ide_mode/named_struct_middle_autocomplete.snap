warning[W10007]: issue with attribute value
  ┌─ tests/move_2024/ide_mode/named_struct_middle_autocomplete.move:1:9
  │
1 │ #[allow(ide_path_autocomplete)]
  │         ^^^^^^^^^^^^^^^^^^^^^ Unknown warning filter 'ide_path_autocomplete'

error[E03010]: unbound field
   ┌─ tests/move_2024/ide_mode/named_struct_middle_autocomplete.move:14:21
   │
14 │         let _tmp2 = _s.b.x;
   │                     ^^^^ Unbound field 'b' in 'a::m::B'

